transformer_engine/pytorch/csrc/extensions/quantizer.cpp

```diff
@@ -103,7 +103,7 @@ std::pair<TensorWrapper, py::object> Float8Quantizer::create_tensor(
   }
   const py::object py_data = rowwise_usage ? py::cast(data) : py::none();
   at::Tensor columnwise_data;
-  bool create_transpose = columnwise_usage && !non_tn_fp8_gemm_supported();
+  bool create_transpose = columnwise_usage && !is_supported_nontn_fp8_gemm();
```

Comment:
@ptrendx I realized that we could also remove the usage of this function for example with the `is_supported_nontn_fp8_gemm` function from common.